simapp/simd/cmd/root.go

```diff
@@ -73,7 +73,7 @@ func init() {
 	authclient.Codec = encodingConfig.Marshaler
 
 	rootCmd.AddCommand(
-		genutilcli.InitCmd(simapp.ModuleBasics, simapp.DefaultNodeHome),
+		genutilcli.InitCmd(simapp.ModuleBasics),
```

Comment:
So I removed `simapp.DefaultNodeHome` from most of the `genutilcli` CLI commands. But, on second thought, it seems maybe this is preferable? WDYT?

It seems we have three options WRT to the home flag for genutil CLI commands:

1. Revert to using `simapp.DefaultNodeHome` instead of `""`, OR
2. Use `""` and mark that flag required, OR
3. Turn it into an arg instead of a flag

Thoughts @alessio ?

Reply:
The one with the required does not look bad. But maybe is less user friendly since you always have to set it by default, and probably 98% of the cases you would be using the default one.

Reply:
Yeah so if default use is the common use, seems like we should revert -- i.e. option (1).

Reply:
I would prefer not to mark a flag as a required. Flags leads to a worse UX IMHO. We should have a sane default that users can override via a flag. If it is required then it should be a positional arg.

Reply:
Ok, reverting to DefaultNode and making it work.

Reply:
Ok let's revert to (1) @jgimeno